README.md

```diff
@@ -4,33 +4,13 @@ This repository is the source code for [Kong](https://github.com/Kong/kong)'s we
 
 ## Requirements
 
-- [npm](https://www.npmjs.com/)
-- [Bundler](http://bundler.io/)
-- [Ruby](https://www.ruby-lang.org) (>= 2.0, < 2.3)
-- [Python](https://www.python.org) (>= 2.7.X, < 3)
+- [docker]
 
-## Install
+## Develop Locally
 
 >
 ```bash
-gem install bundler
-npm install
-```
-
-## Running locally
-
->
-```bash
-npm start
-```
-
-## Deploying
-
-This will deploy to GitHub pages:
-
->
-```bash
-npm run deploy
```

Comment:
I'm not sure we should remove these instructions - they work for many users - maybe you can add your Docker instructions as _an_ option, rather than making it the _only_ choice?

Reply:
mostly for simplicity but I can leave both

npm, ruby, rvm, bundler, grunt is quite the list of prerequisites and gotta get the versions right :(